netket/optimizer/qgt/qgt_jacobian_dense.py

```diff
@@ -37,11 +37,11 @@ def QGTJacobian(vstate, *, mode, rescale_shift=False) -> "QGTJacobianT":
         rescale_shift,
     )
 
-    return QGTJacobianT(O=O, scale=rescale_shift)
+    return QGTJacobianDenseT(O=O, scale=scale, **kwargs)
```

Comment:
This is needed so `diag_shift` can be passed to the constructor, might get more thing to pass through here in due course...